Dockerfile

```diff
@@ -1,10 +1,21 @@
-FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.0
+ARG APP_INSIGHTS_AGENT_VERSION=2.5.0-BETA.3
+
+# Build image
+
+FROM busybox as downloader
+
+RUN wget -P /tmp https://github.com/microsoft/ApplicationInsights-Java/releases/download/2.5.0-BETA.3/applicationinsights-agent-2.5.0-BETA.3.jar
+
+# Application image
+
+FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-debug-1.0
```

Comment:
I guess we need debug version of image for the AI?

Reply:
My mistake, it should not be there, my mistake.

I put it there for troubleshooting times.

Will remove.

Reply:
Done